### ISSUE TITLE
PSRule not finding rules in current path by default

### ISSUE THREAD

**Issue by BernieWhite** [BernieWhite/PSRule#256]
**Description of the issue**

PSRule doesn't find rules in the current working path with `-Path .\` omitted. Previously `Invoke-PSRule`  or `Get-PSRule` will automatically find rules in the current working without requiring the `-Path` parameter.

**To Reproduce**

Steps to reproduce the issue:

```powershell
cd .\docs\scenarios\fruit\;
Get-PSRule;
```
No results returned.

```powershell
cd .\docs\scenarios\fruit\;
Get-PSRule -Path .\
```

Returns:

```text
RuleName                            ModuleName                 Synopsis
--------                            ----------                 --------
isFruit                                                        An example rule
```

**Expected behaviour**

PSRule should detect `.Rule.ps1` files in the current path by default when `-Path` and `-Module` are not specified.

**Error output**

```text
VERBOSE: [Get-PSRule]::BEGIN
VERBOSE: [New-PSRuleOption] BEGIN::
VERBOSE: Attempting to read: C:\removed\docs\scenarios\fruit
VERBOSE: [New-PSRuleOption] END::
VERBOSE: [Get-PSRule] -- Could not find any .Rule.ps1 script files in the path
VERBOSE: [Get-PSRule]::END
```

**Module in use and version:**

- Module: PSRule
- Version: **0.8.0-B190742**

Captured output from `$PSVersionTable`:

```text
Name                           Value
----                           -----
PSVersion                      6.2.1
PSEdition                      Core
GitCommitId                    6.2.1
OS                             Microsoft Windows 10.0.18362
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Additional context**

The issue does not occur on v0.7.0.
